src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1755,6 +1756,16 @@ async function createWidget(widget: any, viewId: string): Promise<Widget | null>
         case 'popup':
             newWidget = await setPopup(widget, viewId) as Popup;
             break;
+        case 'measurement':
+            newWidget = new Measurement({
+                view: view,
+                activeTool: widget.activeTool,
+                areaUnit: widget.areaUnit,
+                linearUnit: widget.linearUnit,
+                label: widget.label,
+                icon: widget.icon,
+            });
```

Comment:
I think this will break if you don't set all the values in the sample. The pattern I suggest is `activeTool: widget.activeTool ?? undefined`. This prevents setting `null` values in arcgis, which are sometimes not expected or handled.